openhcl/underhill_attestation/src/igvm_attest/ak_cert.rs

```diff
@@ -4,31 +4,45 @@
 //! The module for `AK_CERT_REQUEST` request type that supports parsing the
 //! response.
 
-use crate::igvm_attest::Error;
 use crate::protocol::igvm_attest::get::IgvmAttestAkCertResponseHeader;
 use crate::protocol::igvm_attest::get::AK_CERT_RESPONSE_HEADER_VERSION;
+use thiserror::Error;
 use zerocopy::FromBytes;
 
+#[derive(Debug, Error)]
+pub enum AkCertError {
+    #[error("AK cert response size is too small to parse")]
```

Comment:
where/how is this error message displayed?

Reply:
This is displayed when the error is printed.  See thiserror crate.